### DISCUSSION TITLE
Are we ready to start implementing new features?

### DISCUSSION THREAD

**Discussion by sudocanttype** [Bugswriter/tuxi#142] [General]
Earlier we said to hold off new features until we reached a stable main version, and the current main version seems to work without bugs. Is it time? 

Also, if necessary, I can write more test cases for every new feature to be added, to ensure stability.

**Comment by BeyondMagic** [comment]
Yes? But probably fix the current bugs first in the main is a better thing to do, #127 and #112.

**Comment by Genghius** [comment]
*develop branch-breaking PTSD intensifies.*

**Reply by sudocanttype** [reply]
In theory we can't fck it up thaaaaat badly... right? As long as each PR has multiple people try it, and I keep on writing more and more tests, we should be fine.

**Reply by PureArtistry** [reply]
if there's anything you need help with on the new features / bugs front just let me know.
I've got plenty of free time at the moment and this is a fun little project.
